src/reports/suite.js

```diff
@@ -0,0 +1,90 @@
+export default class SuiteReport {
+	constructor( name, parentSuite ) {
+		this.name = name;
+		this.fullName = parentSuite ? parentSuite.fullName.concat( name ) : [];
```

Comment:
Should this have a space or other delimiter between the parent name and this report's name?